authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc

```diff
@@ -0,0 +1,90 @@
+
+By setting different values for the `credentialsMode` parameter in the `install-config.yaml` file, the CCO can be configured to operate in several different modes. If no mode is specified, or the `credentialsMode` parameter is set to an empty string (`""`), the CCO operates in its default mode.
+
+== Modes
+
+By setting different values for the `credentialsMode` parameter in the `install-config.yaml` file, the CCO can be configured to operate in _mint_, _passthrough_, or _manual_ mode. These options provide transparency and flexibility in how the CCO uses cloud credentials to process `CredentialsRequest` CRs in the cluster, and allow the CCO to be configured to suit the security requirements of your organization. Not all CCO modes are supported for all cloud providers.
+
+* **xref:cco-mode-mint.adoc[Mint]**: In mint mode, the CCO uses the provided admin-level cloud credential to create new credentials for components in the cluster with only the specific permissions that are required.
++
+[NOTE]
+====
+Mint mode is the default and recommended best practice setting for the CCO to use.
+====
+
+* **xref:cco-mode-passthrough.adoc[Passthrough]**: In passthrough mode, the CCO passes the provided cloud credential to the components that request cloud credentials.
+
+* **xref:cco-mode-manual.adoc[Manual]**: In manual mode, a user manages cloud credentials instead of the CCO.
+
+** **xref:cco-mode-sts.adoc[Manual with AWS STS]**: In manual mode, you can configure an AWS cluster to use Amazon Web Services Secure Token Service (AWS STS). With this configuration, the CCO uses temporary credentials for different components.
+//Tech Preview only in 4.7, needs disclaimer. Move after mint when feature is GA
++
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+.CCO mode support matrix
+[cols="<.^2,^.^1,^.^1,^.^1"]
+|====
+|Cloud provider |Mint |Passthrough |Manual
+
+|Amazon Web Services (AWS)
+|X
+|X
+|X
+
+
+|Microsoft Azure
+|X
+|X
+|X
+
+|Google Cloud Platform (GCP)
+|X
+|X
+|X
+
+|{rh-openstack-first}
+|
+|X
+|
+
+|{rh-virtualization-first}
+|
+|X
+|
+
+|VMware vSphere
+|
+|X
+|
+
+|====
+
+== Default behavior
+For platforms on which multiple modes are supported (AWS, Azure, and GCP), when the CCO operates in its default mode, it checks the provided credentials dynamically to determine for which mode they are sufficient to process `CredentialsRequest` CRs.
+
+By default, the CCO determines whether the credentials are sufficient for mint mode, which is the preferred mode of operation, and uses those credentials to create appropriate credentials for components in the cluster. If the credentials are not sufficient for mint mode, it determines whether they are sufficient for passthrough mode. If the credentials are not sufficient for passthrough mode, the CCO cannot adequately process `CredentialsRequest` CRs.
```

Comment:
Is this done during installation, as the following paragraph suggests?

Reply:
Tracking in [OSDOCS-1817](https://issues.redhat.com/browse/OSDOCS-1817)